static/res/main.js

```diff
@@ -0,0 +1,89 @@
+var username = "";
+
+var nameField;
+var gongElement;
+
+var storedUname = localStorage.getItem("username");
+
+function init(){
+    gongElement = document.getElementById("gongen");
+    nameField = document.getElementById("name");
+
+    if(storedUname != null){
+        setUsername(storedUname);
+    }
+
+    gongElement.addEventListener("click", function(e){
+        console.log("GONG!");
```

Comment:
Ikke alle nettlesere har støtte for `console` når utvikler verktøy ikke er installert.